packages/react/src/modules/in-app-message/components/Banner/styles.css

```diff
@@ -0,0 +1,54 @@
+.iam-banner {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #ffffff;
+  padding: 16px 24px;
+  border-radius: 8px;
+  border: 0.5px solid #e0e1e6;
```

Comment:
What about bringing in telegraph tokens?

Reply:
Will take a look and doing that!